### ISSUE TITLE
Changing colors in the chart

### ISSUE THREAD

**Issue by ursky** [marbl/Krona#14]
I am trying to use kronatools to produce publication-quality figures, but running into a serious issue with the colors. At the first level of taxonomy (archaea, bacteria, or eukarya) the colors are chosen based on abundance (eg the most abundant will become red). But this produces an issue if I am making a figure to compare two populations because now the same group can be colored differently (eg: bacteria labeled red in fig1 but green in fig2) - I need a way to fix this so that the figures can be easily compared.

tldr: I would like to be able to change the colors on the krona chart without resorting to brute force (like paint...).

Thank you!

**Comment by ondovb** [comment]
The colors will stay consistent for each taxon if you import both samples at the same time (e.g. `ktImportTaxonomy sample1.txt sample2.txt). You can then switch between the samples in the chart to take snapshots. Every sample will be colored by the abundances from the first sample given.

**Comment by ursky** [comment]
Brilliant! Thank you!
